yapapi/executor/__init__.py

```diff
@@ -146,12 +146,15 @@ def __init__(
         self._driver = driver.lower() if driver else DEFAULT_DRIVER
         self._network = network.lower() if network else DEFAULT_NETWORK
         self._stream_output = True
-        self._strategy = (
+        if not strategy:
+            strategy = LeastExpensiveLinearPayuMS(
+                max_fixed_price=Decimal("1.0"),
+                max_price_for={com.Counter.CPU: Decimal("0.2"), com.Counter.TIME: Decimal("0.1")},
+            )
             # The factor 0.5 below means that an offer for a provider that failed to confirm
             # the last agreement proposed to them will have it's score multiplied by 0.5.
-            strategy
-            or DecreaseScoreForUnconfirmedAgreement(LeastExpensiveLinearPayuMS(), 0.5)
-        )
+            DecreaseScoreForUnconfirmedAgreement(strategy, 0.5)
```

Comment:
hmm, we're changing the logic here... maybe we shouldn't apply `DecreaseScoreForUnconfirmedAggreement` to user-supplied strategies ... ?